runtime/kusama/src/weights/pallet_staking.rs

```diff
@@ -44,155 +44,155 @@ use sp_std::marker::PhantomData;
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	fn set_payee() -> Weight {
-		(12_131_000 as Weight)
+		(11_884_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn set_controller() -> Weight {
-		(26_296_000 as Weight)
+		(26_010_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(3 as Weight))
 			.saturating_add(T::DbWeight::get().writes(3 as Weight))
 	}
 	fn set_validator_count() -> Weight {
-		(1_995_000 as Weight)
+		(2_018_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn force_no_eras() -> Weight {
-		(2_311_000 as Weight)
+		(2_321_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn force_new_era() -> Weight {
-		(2_271_000 as Weight)
+		(2_316_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn force_new_era_always() -> Weight {
-		(2_366_000 as Weight)
+		(2_290_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn set_invulnerables(v: u32, ) -> Weight {
-		(2_391_000 as Weight)
+		(2_224_000 as Weight)
 			// Standard Error: 0
-			.saturating_add((35_000 as Weight).saturating_mul(v as Weight))
+			.saturating_add((36_000 as Weight).saturating_mul(v as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn force_unstake(s: u32, ) -> Weight {
-		(58_957_000 as Weight)
+		(58_754_000 as Weight)
 			// Standard Error: 1_000
-			.saturating_add((2_585_000 as Weight).saturating_mul(s as Weight))
+			.saturating_add((2_534_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(8 as Weight))
 			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(s as Weight)))
 	}
 	fn cancel_deferred_slash(s: u32, ) -> Weight {
-		(5_925_666_000 as Weight)
+		(5_931_430_000 as Weight)
 			// Standard Error: 391_000
-			.saturating_add((34_771_000 as Weight).saturating_mul(s as Weight))
+			.saturating_add((34_745_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn payout_stakers_dead_controller(n: u32, ) -> Weight {
-		(114_424_000 as Weight)
-			// Standard Error: 17_000
-			.saturating_add((49_957_000 as Weight).saturating_mul(n as Weight))
+		(129_521_000 as Weight)
+			// Standard Error: 18_000
+			.saturating_add((49_353_000 as Weight).saturating_mul(n as Weight))
 			.saturating_add(T::DbWeight::get().reads(11 as Weight))
 			.saturating_add(T::DbWeight::get().reads((3 as Weight).saturating_mul(n as Weight)))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(n as Weight)))
 	}
 	fn payout_stakers_alive_staked(n: u32, ) -> Weight {
```

Comment:
This is the function which where this constant is used.

Did you check how much weight this function is when N is 256?

Reply:
And that it fits into any of our block limits or anything like that?

Reply:
Added a test to ensure this will never exceed 50% of the block weight. With 256, it'll take 0.06 of the block weight.